### ISSUE TITLE
soup don't close at the end of extraction

### ISSUE THREAD

**Issue by jeg139** [andrew-mi/Soup#15]
When extraction finished, soup don't close automaticaly or have a button to close.

**Comment by andrew-mi** [comment]
Hi @jeg139,
There is a close button in the top right hand corner.

**Comment by andrew-mi** [comment]
It is disabled whilst the program is working, but as soon as extraction is finished it is enabled.

**Comment by jeg139** [comment]
Yes I saw it, and found how to close. Sorry my english is not the best, I'm not a native english speaker. Soup look buggy, is not, but for normal user, when the extraction finish, you just see the program with blue progress bar moving and no end message. I think add a terminated message or a close button in the bottom right corner at the end of extraction may things more clear for end user. Thanks for your time and soup work well and is fast i like it.
